### ISSUE TITLE
Add regression test suite

### ISSUE THREAD

**Issue by fnothaft** [bigdatagenomics/adam#1407]
After #1402, I'm convinced that we can't wait on having a regression test runner anymore. I'm thinking that we add regression tests for:

- [ ] MarkDuplicates vs. Picard (and Samtools and SAMBAMBA and SamBlaster?)
- [ ] Sort reads and save to single file BAM/CRAM
- [ ] FlagStat vs Samtools (and SAMBAMBA)
- [ ] BQSR and RealignIndels vs GATK (?)
- [ ] VCF ingest and export
- [ ] VCF sort
- [ ] VCF annotation ingest
- [ ] Ingest and export (to single file) for all feature file formats
- [ ] Joins

Thoughts?

**Comment by heuermh** [comment]
Closing as WontFix
